### PR TITLE
implement Home page

### PR DESCRIPTION
**Pull request by aEricLiao**
- Implement Home page
- Add a alias '@src' that point to the root "src" path
- add Roboto font

image

### CODE REVIEW

File: ui/index.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <title>Lucky Draw dev</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,300&display=swap" rel="stylesheet">
```
Comment (aStanleyLiang) [inline ui/index.html:7]: Nice 👍

File: ui/src/containers/Layout/Layout.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import 'Src/common/variables.scss';
```
Comment (aStanleyLiang) [inline ui/src/containers/Layout/Layout.scss]: path name is case-sensitive in specific environment. probably correct this with alias I saw you used in another .scss file:
```
 @import '@src/common/variables.scss';
```

**Review by aStanleyLiang — CHANGES_REQUESTED** [review]
overall looks good, minor change is required

**Review by aMichaelPoernomo — APPROVED** [review]
subarashii

File: ui/src/containers/Layout/Layout.scss (hunk 2 above)
Comment (aMichaelPoernomo) [reply ui/src/containers/Layout/Layout.scss]: looks like we will have 2 duplicated variable directory, `src/common/variables.scss` and `src/style/common.scss`

Comment (aEricLiao) [reply ui/src/containers/Layout/Layout.scss]: oops, I'll keep the`style/common.css` 👽

Comment (aEricLiao) [reply ui/src/containers/Layout/Layout.scss]: fix in https://github.com/aStanleyLiang/luckydraw/pull/14/commits/5a26b66564d015ef77ad5fcce19b6fdbb2bc745e thx~

Comment (aStanleyLiang) [reply ui/src/containers/Layout/Layout.scss]: @aMichaelPoernomo  nice catch

**Review by aStanleyLiang — APPROVED** [review]

**Review by aYishinHo — APPROVED** [review]

**Pull request merged**